src/TwigComponent/src/CVA.php

```diff
@@ -53,6 +53,14 @@ public function resolve(array $recipes): string
         }
 
         foreach ($recipes as $recipeName => $recipeValue) {
+            if (isset($this->variants[$recipeName]) && (\is_string($this->variants[$recipeName]) || (\is_array($this->variants[$recipeName]) && array_is_list($this->variants[$recipeName])))) {
+                if (true === $recipeValue) {
+                    $classes .= ' '.implode(' ', (array) $this->variants[$recipeName]);
+                }
+
+                continue;
+            }
+
             if (!isset($this->variants[$recipeName][$recipeValue])) {
```

Comment:
Hmm.

Reply:
There is something i don't understand in the algorithm will ask you questions about this, but don"t wait for me in this PR, that's 100% implementation and can be handled way after :)